core/src/commands/moderation.cpp

```diff
@@ -170,11 +170,15 @@ void AOClient::cmdHelp(int argc, QStringList argv)
 {
     if(argc > 1) {
         sendServerMessage("Too many arguments. Please only use the command name.");
+        return;
     }
 
     QString l_command_name = argv[0];
     ConfigManager::help l_command_info = ConfigManager::commandHelp(l_command_name);
-    sendServerMessage("==Help==\n" +l_command_info.usage + "\n" + l_command_info.text);
+    if (l_command_info.usage.isEmpty() || l_command_info.text.isEmpty()) // my picoseconds :(
```

Comment:
what does this mean

Reply:
I was just talking with Salanto about it being slightly inefficient to check for both when a check for one of those two would probably be fine for this case, but he said it's mere picoseconds and doesn't really matter.